main.java

```diff
@@ -0,0 +1,5 @@
+public class Main {
+  public static void main(String[] args) {
+    System.out.println("Hello, World!");
+  }
```

Comment:
Add a function to print the first n fibonacci numbers

Reply:
done